compiler/rustc_mir/src/util/borrowck_errors.rs

```diff
@@ -68,9 +68,10 @@ impl<'cx, 'tcx> crate::borrow_check::MirBorrowckCtxt<'cx, 'tcx> {
             err.span_label(
                 new_loan_span,
                 format!(
-                    "mutable borrow starts here in previous \
-                     iteration of loop{}",
-                    opt_via
+                    "mutable variable {}{} is already borrowed in the previous iteration of the loop{}",
```

Comment:
I find this error confusing: the issue is not that _x_ is mutable, rather that it was mutably _borrowed_. How about something like this?

```suggestion                    "{}{} was already mutably borrowed in the previous iteration of the loop{}",
```